internal/registry/credentialrequest/rest_test.go

```diff
@@ -27,10 +27,20 @@ import (
 	"go.pinniped.dev/internal/testutil"
 )
 
+// Assert that our *REST implements all the optional interfaces that we expect it to implement.
+var _ interface {
+	rest.Creater
+	rest.NamespaceScopedStrategy
+	rest.Scoper
+	rest.Storage
+	rest.CategoriesProvider
+} = (*REST)(nil)
```

Comment:
Could you put this in the `rest.go` file so that its a build failure even when tests are not run?